inference-engine/tests/functional/shared_test_classes/src/single_layer/convolution.cpp

```diff
@@ -41,23 +46,29 @@ std::string ConvolutionLayerTest::getTestCaseName(testing::TestParamInfo<convLay
 
 void ConvolutionLayerTest::SetUp() {
     convSpecificParams convParams;
-    std::vector<size_t> inputShape;
-    auto netPrecision   = InferenceEngine::Precision::UNSPECIFIED;
-    std::tie(convParams, netPrecision, inPrc, outPrc, inLayout, outLayout, inputShape, targetDevice) =
+    std::vector<std::pair<size_t, size_t>> inputShape;
+    std::vector<InferenceEngine::SizeVector> targetShapes;
+    std::tie(convParams, netPrecision, inPrc, outPrc, inLayout, outLayout, inputShape, targetShapes, targetDevice) =
         this->GetParam();
-    ngraph::op::PadType padType;
-    InferenceEngine::SizeVector kernel, stride, dilation;
-    std::vector<ptrdiff_t> padBegin, padEnd;
-    size_t convOutChannels;
+    for (auto&& targetShape : targetShapes) {
+        targetStaticShapes.emplace_back(targetShape);
+    }
+    inputDynamicShape = FuncTestUtils::PartialShapeUtils::vec2partialshape(inputShape, targetStaticShapes[0]);
     std::tie(kernel, stride, padBegin, padEnd, dilation, convOutChannels, padType) = convParams;
+
+    setTargetStaticShape(targetStaticShapes[0]);
+    makeConvolutionFunction();
```

Comment:
please rename the function according the code style and move it to the [builders](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/ngraph_helpers/ngraph_functions/include/ngraph_functions/builders.hpp) and [here](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/ngraph_helpers/ngraph_functions/src/convolution.cpp)

Reply:
It's changed to makeConvolution(), and could you let me know if you have any other suggestion or comment?